src/contentParser.js

```diff
@@ -37,6 +37,15 @@ export default function contentParser(
     const subdir = wordPressUrlParsed.path();
     return path.replace(subdir, '/');
   };
+	
+	const getAnchor = (url) => {
```

Comment:
@DogByteMarketing Can you pull the formatting in line with the rest of the package?

Reply:
That's the weird thing, my code doesn't show up like it's showing on github

Reply:
I copied code issued PR again